src/m365/spo/commands/file/file-copy.ts

```diff
@@ -279,28 +186,12 @@ class SpoFileCopyCommand extends SpoCommand {
     });
   }
 
-  /**
-   * Combines base and relative url considering any missing slashes
-   * @param baseUrl https://contoso.com
-   * @param relativeUrl sites/abc
-   */
-  private urlCombine(baseUrl: string, relativeUrl: string): string {
-    // remove last '/' of base if exists
-    if (baseUrl.lastIndexOf('/') === baseUrl.length - 1) {
-      baseUrl = baseUrl.substring(0, baseUrl.length - 1);
-    }
-
-    // remove '/' at 0
-    if (relativeUrl.charAt(0) === '/') {
-      relativeUrl = relativeUrl.substring(1, relativeUrl.length);
-    }
-
-    // remove last '/' of next if exists
-    if (relativeUrl.lastIndexOf('/') === relativeUrl.length - 1) {
-      relativeUrl = relativeUrl.substring(0, relativeUrl.length - 1);
+  public cancel(): CommandCancel {
```

Comment:
As we removed immersive mode in v3 we no longer need to support cancelling the command, because closing the process will also close any pending requests. We can also remove it when we move off of Vorpal. No harm done :)